extensions/amp-next-page/1.0/page.js

```diff
@@ -279,16 +279,16 @@ export class HostPage extends Page {
    * @param {{ url: string, title: string, image: string }} meta
    * @param {!PageState} initState
    * @param {!VisibilityState} initVisibility
-   * @param {!Document} initDoc
+   * @param {!Document} doc
```

Comment:
How did this change meaning?

Reply:
It did not but it makes more sense to call it doc than `initDoc` since the document won't change over time (contrary to `initiVisibility` and `initState`)

Reply:
Thanks for the explanation!